modules/openapi-generator/src/main/resources/C-libcurl/CMakeLists.txt.mustache

```diff
@@ -27,6 +28,7 @@ else()
 endif()
 
 set(pkgName "{{projectName}}")
+set(VERSION 0.0.1)
```

Comment:
And add an annotation to tell user the "VERSION" can be changed by some other file.

Reply:
Done